tests/queries/0_stateless/03132_pretty_format_break_line.sql

```diff
@@ -0,0 +1,10 @@
+DROP TABLE IF EXISTS t_break_line;
+
+CREATE TABLE t_break_line (id UInt64, value String) ENGINE=MergeTree ORDER BY id;
+
+INSERT INTO t_break_line VALUES(0, 'hello\nworld');
+
+SELECT * FROM t_break_line FORMAT PrettyCompactNoEscapes SETTINGS output_format_pretty_row_numbers = 0;
+SELECT * FROM t_break_line FORMAT PrettyCompactNoEscapes;
```

Comment:
Would be good to add tests for some other Pretty Formats (PrettyMonoBlock, PrettyCompact, Pretty, PrettySpace etc)